lua/nvim-tree/legacy.lua

```diff
@@ -190,6 +190,27 @@ local function refactored(opts)
       end
     end
   end
+
+  if opts.update_to_buf_dir then
+    utils.table_create_missing(opts, "hijack_directories")
+    if opts.hijack_directories.enable == nil then
+      opts.hijack_directories.enable = opts.update_to_buf_dir.enable
+    end
+    if opts.hijack_directories.auto_open == nil then
+      opts.hijack_directories.auto_open = opts.update_to_buf_dir.auto_open
+    end
+    opts.update_to_buf_dir = nil
+  end
+
+  if opts.view then
+    if opts.view.auto_resize ~= nil then
+      utils.table_create_missing(opts, "actions.open_file")
+      if opts.actions.open_file.resize_window == nil then
+        opts.actions.open_file.resize_window = opts.view.auto_resize
+      end
+      opts.view.auto_resize = nil
+    end
+  end
```

Comment:
i think we can remove this though. A better strategy for the plugin stability is needed. I'll think about this and propose a solution that could work :)

Reply:
Yes. This is very hacky.

However... we could just remove most of this code in a few months, as users will have migrated.